canonical-web-and-design/ubuntu.com
# Update renamed Vanilla patterns

## Done

- renamed `.p-code-snippet` class to `.p-code-copyable`
- renamed `.p-list-step` and related classes to `.p-stepped-list`
  - also introduced a temporary fix for a Vanilla bug that appears when`p` tags have the `.p-stepped-list__title` class, being addressed [here](https://github.com/canonical-web-and-design/vanilla-framework/issues/2489)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- On the following list of pages, ensure that the stepped lists and copyable code elements are not broken:
  - [x] /about/release-cycle
  - [x] /download/iot/installation-media
  - [x] /download/iot/intel-iei-tank-870
  - [x] /download/iot/intel-joule-desktop
  - [x] /download/iot/intel-joule
  - [x] /download/iot/intel-nuc-desktop
  - [x] /download/iot/intel-nuc
  - [x] /download/iot/kvm
  - [x] /download/iot/orange-pi-zero
  - [x] /download/iot/qualcomm-dragonboard-410c
  - [x] /download/iot/raspberry-pi-2-3-core
  - [x] /download/iot/raspberry-pi-2-3
  - [x] /download/iot/raspberry-pi-compute-module-3
  - [x] /download/iot/samsung-artik-5-10
  - [x] /download/iot/up-squared-iot-grove-server
  - [x] /download/server/provisioning
  - [x] /embedded
  - [x] /esm
  - [x] /internet-of-things/robotics
  - [x] /kubeflow/install
  - [x] /kubernetes/docs/quickstart
  - [x] /kubernetes/install
  - [x] /kubernetes/managed
  - [x] /livepatch
  - [x] /openstack/install
  - [x] /openstack/managed


## Issue / Card

Fixes #5646
